modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model/tests

```diff
@@ -25,7 +25,7 @@
     input = 'rayleigh-bernard-two-phase.i'
     csvdiff = 'rayleigh-bernard-two-phase_CSV.csv'
     requirement = 'The system shall be able solve two phase Rayleigh Bernard convection with the mixture model.'
-    rel_err = 5.0E-3
+    rel_err = 5.0E-1
```

Comment:
should we tighten this instead?
we just regolded so we should be passing the new test. and with the slip velocity now in use, maybe the test is not stochastic anymore?

Reply:
Add to the test requirement